utils.go

```diff
@@ -57,3 +58,30 @@ func parseImageDestination(c *cli.Context, name string) (types.ImageDestination,
 	}
 	return nil, fmt.Errorf("Unrecognized image reference %s", name)
 }
+
+func parseDockerImage(img, certPath string, tlsVerify bool) (types.Image, error) {
+	s, err := newDockerImageSource(img, certPath, tlsVerify)
+	if err != nil {
+		return nil, err
+	}
+	return &dockerImage{src: s}, nil
+}
```

Comment:
I think this really belongs into `docker_image.go`; it is a constructor for `dockerImage`.